### PR TITLE
test output of csv

### LINKED ISSUE #179: Separate code from data (Python folder)
I think we shoud put the scripts code in a separate folder, so all scripts would be in a separate folder, we can name it src, code, ...

image

### PR DESCRIPTION
**Pull request by bensialih**
test conversion script from json to csv

### CODE REVIEW

**Comment by seddik** [comment]
could you trim this, and make the script replaces quotes then trim, not trim then replace quotes.


image

**Comment by seddik** [comment]
@bensialih  there is the double space also we have 34 cases, so please add it to the script after removing the quotes
1- remove quotes
2- replace double spaces
3- trim (Left & Right)

image

**Review by seddik — CHANGES_REQUESTED** [review]
@bensialih for the first file  it looks GOOD 👍✅ `csv/algeria_cities_export.csv]`

for the other ones they are made with the old script I think
1- I think we need to get rid of the double quotes "
2- there are places with **double space**

**Comment by bensialih** [comment]
@seddik  I will have a look this hopefully on Wednesday or next weekend. Hopefully next weekend I will make some more changes

File: csv/algeria_postcodes_export.csv [hunk 1]
```diff
@@ -0,0 +1,3941 @@
...
+03037,أنفوس,Anfous,أنفوس,Anfous,71,الغيشة,El Ghicha,الغيشة,El Ghicha,03,الأغواط,Laghouat
+03038,حاج مشاري,Hadj Mecheri,حاج مشاري,Hadj Mecheri,74,الحاج مشري,Hadj Mechri,بريدة,Brida,03,الأغواط,Laghouat
+03040,الحويطة,El Houita,الحويطة,El Houita,72,الحويطة,El Haouaita,عين ماضي,Ain Madhi,03,الأغواط,Laghouat
+03041,وادي مزي,Oued M'Zi,وادي مزي,Oued M'Zi,81,وادي مزي,Oued M'zi,وادي مرة,Oued Morra,03,الأغواط,Laghouat
+03042,لالماية,Lalmaya,لالماية,Lalmaya,86,تاجرونة,Tadjrouna,عين ماضي,Ain Madhi,03,الأغواط,Laghouat
+03043,وادي الطويل,Oued Touil,وادي الطويل,Oued Touil,73,قلتة سيدي سعد,Gueltat Sidi Saad,قتلة سيدي سعيد,Gueltat Sidi Saad,03,الأغواط,Laghouat
+03044,حسيان الذيب,Hassiane Dhib,حسيان الذيب,Hassiane Dhib,73,قلتة سيدي سعد,Gueltat Sidi Saad,قتلة سيدي سعيد,Gueltat Sidi Saad,03,الأغواط,Laghouat
+03046,بوزباير,Bouzbaeir,بوزباير,Bouzbaeir,76,حاسي الرمل,Hassi R'mel,حاسي الرمل,Hassi R'mel,03,الأغواط,Laghouat
+03047,أفلو - زاوي سعيد,Aflou Zaoui Said,أفلو - زاوي سعيد,Aflou Zaoui Said,64,أفلو,Aflou,أفلو,Aflou,03,الأغواط,Laghouat
+03048,"الأغواط - ""DCA""",Laghouat D C A,"الأغواط - ""DCA""",Laghouat Dca,79,الأغواط,Laghouat,الأغواط,Laghouat,03,الأغواط,Laghouat
```
Comment (bensialih) [inline csv/algeria_postcodes_export.csv:208]: I think things like these double quotes are present in the Json payload too.
These are put in when double quotes are part of the entry and should be looked into as possible flaws for that entry.

Comment (bensialih) [reply csv/algeria_postcodes_export.csv:208]: I think we should stop here and not do any more formatting. I worry we are experiencing mission creep.

I think maybe formatting should be its own separate objective.

Comment (bensialih) [reply csv/algeria_postcodes_export.csv:208]: quotes pertaining to preceding or trailing spaces disappear when these spaces are removed but as stayed above, other scenarios may need to be addressed.

**Pull request merged**